server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -574,7 +575,9 @@ public void apply(Settings value, Settings current, Settings previous) {
             FsHealthService.ENABLED_SETTING,
             FsHealthService.REFRESH_INTERVAL_SETTING,
             FsHealthService.SLOW_PATH_LOGGING_THRESHOLD_SETTING,
-            IndexingPressure.MAX_INDEXING_BYTES)));
+            IndexingPressure.MAX_INDEXING_BYTES,
+            MasterTaskThrottler.ENABLE_MASTER_THROTTLING,
+            MasterTaskThrottler.THRESHOLD_SETTINGS)));
```

Comment:
[Old PR Comment:](https://github.com/opensearch-project/OpenSearch/pull/484#discussion_r611459098)

We will need to document these new settings and also label the PR accordingly for minor version, it will be available in.

Reply:
Yes, once we finalized the documentation strategy will update document. I have updated this in SIM to keep track of documentation. Will keep that issue open until documentation is not added.

Don't have permission to add lebel in PR.